src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
I'm guessing this test fails? I'd replace it by a similar dummy placeholder test that does not fail and add a TODO comment to write some actual tests.